contrib/opencensus-ext-stackdriver/README.rst

```diff
@@ -35,20 +35,19 @@ This example shows how to report the traces to Stackdriver Trace:
     pip install google-cloud-trace
     pipenv install google-cloud-trace
 
-By default, traces are exported synchronously, which introduces latency during
-your code's execution. To avoid blocking code execution, you can initialize
-your exporter to use a background thread.
+By default, traces are exported asynchronously, to avoid latency during
```

Comment:
nit: avoid -> reduce

I doubt even the async exporter can completely avoid latency.